### ISSUE TITLE
Better support for singular tests

### ISSUE THREAD

**Issue by miro-ur** [fal-ai/fal#497]
**Describe the bug**
Fal object model does not seem to support singular tests. Getting various errors with dbt project that uses custom singular tests (standalone sql tests).

```py
from fal import FalDbt
faldbt = FalDbt(profiles_dir='.dbt', project_dir='/workspace')
print(faldbt.tests[0])
# Getting AttributeError: 'DbtTest' object has no attribute 'column'
for t in faldbt.tests:
    print(t.name)
# Geeting a AttributeError: 'ParsedSingularTestNode' object has no attribute 'test_metadata'
```

**Your environment**
```bash
# fal --version
fal 0.4.1
# dbt --version
Core:
  - installed: 1.2.0
  - latest:    1.2.0 - Up to date!
Plugins:
  - snowflake: 1.2.0 - Up to date!
```
**How to reproduce**
We have a dbt project with generic and custom tests, running `dbt run` and then code above

**Expected behavior**
Expect to be able to retrieve items from tests collection and be able to work with them regardless of type of test

**Actual behavior**
Can not work with tests in fal

**Comment by chamini2** [comment]
Thanks for the report @miro-ur , I will take a look!

**Comment by chamini2** [comment]
@miro-ur , this is released in [0.5.1](https://github.com/fal-ai/fal/releases/tag/v0.5.1). Let us know if any other problem arises! Thanks for the report.
